src/udev/udev-builtin-keyboard.c

```diff
@@ -69,10 +68,12 @@ static int map_keycode(sd_device *dev, int fd, int scancode, const char *keycode
         if (k) {
                 keycode_num = k->id;
         } else {
+                int r;
```

Comment:
Usually in our code `int r` is special. Could you move this at the beginning of the function?

Reply:
Done.